.github/workflows/deploy.yml

```
@@ -0,0 +1,41 @@
+name: Deploy to Netlify
+
+on:
+  push:
+    branches: ['main']
+  workflow_dispatch:
+    inputs:
+      deploy-msg:
+        description: 'Deploy to Production'
+        required: true
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    name: Build and Deploy to Netlify
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3.2.0
+
+      - name: Install modules
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+
+      - name: Deploy to Netlify
+        uses: nwtgck/actions-netlify@v2.0
+        with:
+          publish-dir: './build'
+          deploy-message: 'Deploy from GitHub Actions: ${{ github.event.inputs.deploy-msg }}'
```

I am wondering what's the purpose of `inputs.deploy-msg` here, what about we drop it altogether? I think the message `Deploy from GitHub Actions` is already self-explanatory 

```suggestion
          deploy-message: 'Deploy from GitHub Actions'
```

---

@tyrro this is from the react-templates 😄 Let me remove it